tests/cli/test_get.py

```diff
@@ -0,0 +1,154 @@
+import pytest
+
+from spinta.exceptions import NoPrimaryKeyCandidatesFound, PropertyNotFound
+from spinta.testing.cli import SpintaCliRunner
+from spinta.manifests.tabular.helpers import striptable
+from spinta.testing.tabular import create_tabular_manifest
+
+
+def yaml_content():
+    return """
+---
+_type: datasets/gov/example/City
+_id: 0AF24A60-00A2-4EAB-AEFF-BBA86204BC98
+name: Vilnius
+country:
+  _id: 4689C28B-1C44-4184-8715-16021EE87EAD
+  name: Lietuva
+---
+_type: datasets/gov/example/Country
+_id: 4689C28B-1C44-4184-8715-16021EE87EAD
+name: Lietuva
+---
+    """
+
+
+def yaml_content_with_wrong_ref():
+    return """
+---
+_type: datasets/gov/example/City
+_id: 0AF24A60-00A2-4EAB-AEFF-BBA86204BC98
+name: Vilnius
+country:
+  _id: 4689C28B-1C44-4184-8715-16021EE87EAC
+  name: Lietuva
+---
+_type: datasets/gov/example/Country
+_id: 4689C28B-1C44-4184-8715-16021EE87EAD
+name: Lietuva
+---
+"""
+
+
+def yaml_content_without_field():
+    return """
+---
+_type: datasets/gov/example/City
+_id: 0AF24A60-00A2-4EAB-AEFF-BBA86204BC98
+country:
+  _id: 4689C28B-1C44-4184-8715-16021EE87EAC
+  name: Lietuva
+---
+_type: datasets/gov/example/Country
+_id: 4689C28B-1C44-4184-8715-16021EE87EAD
+name: Lietuva
+---
+"""
+
+
+def test_getall(context, rc, cli: SpintaCliRunner, tmp_path):
+    manifest = striptable('''
+     d | r | b | m  | property         | type         | ref     | source     | access
+     datasets/gov/example              |              |         |            |
+       | data                          | dask/memory  |         |            |
+       |   |                           |         |         |            |
+       |   |   | Country               |         | _id     |            | open
+       |   |   |    | _id              | string  |         | _id        |
+       |   |   |    | name             | string  |         | name       |
+       |   |   |    |                  |         |         |            |
+       |   |   | City                  |         |         |            | open
+       |   |   |    | _id              | string  |         | _id        |
+       |   |   |    | name             | string  |         | name       |
+       |   |   |    | country          | ref     | Country | ..         |
+        """
```

Comment:
Could we format these DSA's correctly as well? And I see a trailing `"""` which should not be here I think? Same for the other two tests as well